src/Shared/Constants.cs

```diff
@@ -149,6 +149,7 @@ internal static class ItemMetadataNames
         internal const string redist = "Redist";
         internal const string resolvedFrom = "ResolvedFrom";
         internal const string destinationSubDirectory = "DestinationSubDirectory";
+        internal const string destinationSubPath = "DestinationSubPath";
```

Comment:
Under what circumstances does the SDK define this instead of `DestinationSubDirectory`?

Reply:
Nuget packages can have copy local assets that need to be available at runtime as opposed to compile time assets that go in the ref sub-folder of the Nuget package. These runtime assets are placed in a sub-folder called runtimes in the Nuget packages. 

There is a DotNet SDK target called ResolvePackageAssets that will add the DestinationSubDirectory metadata to these runtimes assets. Publishing providers need to look at this metadata so that they can then place the runtime assets in the correct subfolder while publishing the app.